Probesys/bileto
# new: Go to top / bottom

## Related issue(s)



https://github.com/Probesys/bileto/issues/236

## Changes



- add a Stimulus controller to scroll to elements
- add a button to the bottom of the pages to scroll to top: it only appears if you scroll enough first
- add a button to the tickets "show" page to scroll to the bottom of the page (i.e. to the editor)

## How to test manually



- open a ticket and add some comments / generate events
- check that "scroll to bottom" scrolls to the editor
- check the button "scroll to top" appears and that clicking on it scroll to the top of the page

## Checklist
